bin/quarry.lua

```diff
@@ -0,0 +1,193 @@
+local radius, keepItems = ...
+radius = tonumber(radius)
+
+if not radius then
+    print('Please specify a side length (number, arg 1)')
+    return
+end
+
+if keepItems == 'true' then
+    keepItems = true
+elseif keepItems == 'false' then
+    keepItems = false
+else
+    print('Please specify if I should keep junk items (e.g. cobblestone) (true/false, arg 2)')
+    return
+end
+
+if not turtle then
+    print('This must be run from a turtle')
+    return
+end
+
+local goodMatches = {
+    'coal',
+    'iron',
+    'copper',
+    'gold',
+    'redstone',
+    'emerald',
+    'lapis',
+    'diamond',
+    'debris',
+    'netherite',
+    'flint'
+}
```

Comment:
should add a mode that takes these as arguments (which I can do)

Reply:
sure, you can do that